tutorials/docs-01-contributing-guide/index.qmd

```diff
@@ -2,17 +2,28 @@
 title: Contributing
 ---
 
-Turing is an open-source project. If you feel that you have relevant skills and are interested in contributing, please get in touch. You can contribute by opening issues on GitHub, implementing things yourself, and making a pull request. We would also appreciate example models written using Turing.
+Turing is an open-source project and is [hosted on GitHub](https://github.com/TuringLang).
+We welcome contributions from the community in all forms large or small: bug reports, feature implementations, code contributions, or improvements to documentation or infrastructure are all extremely valuable.
+We would also very much appreciate examples of models written using Turing.
 
-Turing has a [style guide](#style-guide). Reviewing it before making a pull request is not strictly necessary, but you may be asked to change portions of your code to conform with the style guide before it is merged.
+### How to get involved
 
-### What Can I Do?
+Our outstanding issues are tabulated on our [issue tracker](https://github.com/TuringLang/Turing.jl/issues).
+Closing one of these may involve implementing new features, fixing bugs, or writing example models.

```

Comment:
There are a couple of things here which I think it might be worth thinking about:

1. Turing issues aren't all hosted on Turing.jl, they're scattered around the entire organisation. I don't know if there's a way to view all of them at the same time.

2. If someone wants to work on an issue, it would be good for at least one of us to be aware of that. It's probably a bit intimidating for someone new to directly ping a person, and in any case the person that is pinged might not see it because they're away etc.

   I think it would be good to create [an organisation team](https://github.com/orgs/TuringLang/teams) like `@turinglang/maintainers` or something like that, and say "if you want to take on an open issue, ping this group of people". Then one of us could (for example) assign the issue to them. (This is a pattern I've seen on other projects.)

Reply:
I like this idea of maintainers team.

Also, if you have any ideas for how to keep tabs on what's going on in the repo, I'm looking for better ways. I would ideally like an email notification every time a new issue or PR is opened, but not every time a comment is left in any issue or PR, and I haven't found a way to do it. Either I get unmanageable amounts of spam, or I miss out on important conversations and changes.

Could discuss this at one of the weekly catch-ups.